content/go/concepts/switch/switch.md

```diff
@@ -0,0 +1,120 @@
+---
+Title: 'Switch'
+Description: 'A switch statement allows the selective execution of one of several blocks of code based on the value of an expression.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Comparison'
+  - 'Control Flow'
+  - 'Switch'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+A `switch` statement allows the selective execution of one of several blocks of code based on the value of an expression. It can be thought of as an expanded `if ... then ... else` statement, with many potential `then` clauses.
+
+## Syntax 1
+
+This is the typical `switch` statement syntax:
+
+```pseudo
+switch <expression> {
+  case <value 1>:
+    <statements>
+  case <value 2>:
+    <statements>
+  .
+  .
+  .
+  case <value N>:
+    <statements>
+  default:
+    <statements>
+}
+```
+
+When `<expression>` is equal to the value of one of the `case` statements, the corresponding `<statements>` block is executed. If the `<expression>` matches none of the `case` values, then the optional `default` `<statements>` block is executed.
+
+The `switch` statement in Go differs from other languages like [C++](https://www.codecademy.com/resources/docs/cpp/switch) and [Java](https://www.codecademy.com/resources/docs/java/switch), in that the values used by the `case` statements don't need to be constants, they can be variables or full expressions. Also, unlike in C ++ and Java, Go only executes the selected "case" statement block, so no "break" statement is needed.
```

Comment:
```suggestion
In Go, the `switch` statement differs from other languages like [C++](https://www.codecademy.com/resources/docs/cpp/switch) and [Java](https://www.codecademy.com/resources/docs/java/switch) in that the values used by the `case` statements are not limited to constants. They can also be variables or full expressions. Additionally, Go only executes the selected `case <statements> block, so no "break" statement is needed.
```

> Additionally, Go only executes the selected `case <statements>` block, so no "break" statement is needed. 

Does this mean that Go will keep moving down the program and exit the switch statement after a case is selected?

Reply:
Never mind just confirmed that's the case (yay Go Playground!)